### PR TITLE
#167918913 Setup Babel

### PR DESCRIPTION
**Pull request by Abobos**
**What does this PR do?**

* Setup Babel

**Description of Task to be completed?**

- Install all babel core dependencies
- Create a .babelrc file
- Configure babel to use preset: @babel/preset-env
- Write a JavaScript file using ES6 import syntax
- Run the JavaScript file with @babel/node
- Transpile the Javscript file and output into a build directory

**How should this be manually tested?**

* Clone the repo
* Open terminal
* cd to project root
* run `npm install`
* run  `npm start`
* run `npm run build`
* Test endpoint using postman

**What are the relevant Pivotal Stories?**

[#167918913](https://www.pivotaltracker.com/story/show/167918913)

**Screenshots (if appropriate)**

None

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "@babel/preset-env"
+  ]
+}
```
Comment (biolabalo) [inline .babelrc]: Fix this end of newline

**Comment by dieudonneAwa** [comment]
Hey @Abobos 
After fixing your new line error, I think your PR looks just fine now.

**Review by Wokoro — COMMENTED** [review]
I think we would need `barbel register`and `barbel runtime` for setting up our testing framework e.g mocha to convert our code to ES5 when running tests.

**Review by dieudonneAwa — APPROVED** [review]

**Comment by Abobos** [comment]
@Wokoro Installing @babel/core Installed @babel/register, regenerator-runtime under the hood.. So it will work when you use it.

**Review by biolabalo — APPROVED** [review]

**Pull request merged**